### DISCUSSION TITLE
lost data from fop.saj-electric.com

### DISCUSSION THREAD

**Discussion by TecnoTalarn** [djansen1987/SAJeSolar#89] [General]
![image](https://github.com/user-attachments/assets/a0b70462-11c3-402b-950e-495b4819f9ac)
Hello,
I've been using this integration for a few years now and it worked until a few days ago.
Now, I missed some data.
Some I don't care about, but others I use for statistics.
It seems, however, that this data that we no longer have does exist on the new platform.
Is it planned to change the integration to the new platform?

![image](https://github.com/user-attachments/assets/210d7c31-1032-48fe-801e-7e5f1320e887)
as you can see in the screenshot, since the 22nd of August I have no access to the energy bought, sold or to the battery charge/discharge.

It feels like SAJ is slowly "turning off" the https://fop.saj-electric.com/saj/ platform and letting it die.
thank you for your work.

**Comment by djansen1987** [comment]
There have been some updates. Does it work again?
